src/openforms/formio/api/views.py

```diff
@@ -0,0 +1,75 @@
+import os
+
+from django.conf import settings
+from django.template.defaultfilters import filesizeformat
+from django.utils.translation import gettext_lazy as _
+
+from drf_spectacular.utils import extend_schema
+from rest_framework import status
+from rest_framework.generics import GenericAPIView
+from rest_framework.response import Response
+
+from openforms.api.parsers import MaxFilesizeMultiPartParser
+from openforms.submissions.api.permissions import AnyActiveSubmissionPermission
+from openforms.submissions.api.renderers import JSONOrPlainTextRenderer
+from openforms.submissions.api.serializers import TemporaryFileUploadSerializer
+from openforms.submissions.attachments import clean_mime_type
+from openforms.submissions.models import TemporaryFileUpload
+from openforms.submissions.utils import add_upload_to_session
+
+
+@extend_schema(
+    summary=_("Create temporary file upload"),
+    description=_(
+        'File upload handler for the Form.io file upload "url" storage type.\n\n'
+        "The uploads are stored temporarily and have to be claimed by the form submission "
+        "using the returned JSON data. \n\n"
+        "Access to this view requires an active form submission. "
+        "Unclaimed temporary files automatically expire after {expire_days} day(s). \n\n"
+        "The maximum upload size for this instance is `{max_upload_size}`. Note that "
+        "this includes the multipart metadata and boundaries, so the actual maximum "
+        "file upload size is slightly smaller."
+    ).format(
+        expire_days=settings.TEMPORARY_UPLOADS_REMOVED_AFTER_DAYS,
+        max_upload_size=filesizeformat(settings.MAX_FILE_UPLOAD_SIZE),
+    ),
+)
```

Comment:
The validation error response format needs to be documented explicitly in the API docs - now it's still emitting the standard JSON output.

Reply:
I applied the `responses` dictionary to `extend_schema` in the PR.